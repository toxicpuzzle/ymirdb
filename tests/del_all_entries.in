SET a 1 2 3
SET b 1 2 3
SET c 1 2 3
SET d 10 20 30
PUSH a b
PUSH b c d
LIST ENTRIES
DEL b
DEL a 
DEL b
DEL c
DEL d
LIST ENTRIES
bye 