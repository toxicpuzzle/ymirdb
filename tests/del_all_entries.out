> ok

> ok

> ok

> ok

> ok

> ok

> d [10 20 30]
c [1 2 3]
b [d c 1 2 3]
a [b 1 2 3]

> not permitted

> ok

> ok

> ok

> ok

> no entries

> bye